lib/frinkiac/screencap.rb

```diff
@@ -24,7 +24,7 @@ def meme_url(caption = nil)
       caption = self.caption if caption.nil?
       caption = caption.join("\n") if caption.is_a?(Array)
 
-      "#{SITE_URL}/meme/#{episode}/#{timestamp}.jpg?lines=#{URI.escape caption}"
+      "#{SITE_URL}/meme/#{episode}/#{timestamp}.jpg?b64lines=#{Base64.strict_encode64 caption}"
```

Comment:
Would you mind adding ```require 'base64'``` to the top of the file? I had to add it on my local for this to work.